crates/sui-types/src/messages_consensus.rs

```diff
@@ -67,6 +69,8 @@ pub enum ConsensusTransactionKey {
     // Key must include both id and jwk, because honest validators could be given multiple jwks for
     // the same id by malfunctioning providers.
     NewJWKFetched(Box<(AuthorityName, JwkId, JWK)>),
+    RandomnessDkgMessage(AuthorityName),
```

Comment:
The consensus key is used to de-duplicate consensus messages. Is it the case that each authority only sends one of each of the two Dkg messages per epoch?

Reply:
Yes it is, or at least we only want to use the first one that appears in consensus and can ignore any subsequent ones